src/modules/form/components/DynamicField.tsx

```diff
@@ -258,11 +259,13 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
             id={linkId}
             value={value || ''}
             onChange={onChangeEvent}
-            sx={{
-              width,
-              maxWidth: MAX_INPUT_AND_LABEL_WIDTH,
-              '.MuiInputBase-root': { maxWidth: MAX_INPUT_WIDTH },
+            InputProps={{
+              sx: {
+                width,
+                maxWidth: MAX_INPUT_WIDTH,
```

Comment:
see other comment re: fixed with of phone input

Reply:
For components like phone (and ssn and etc.), should we still accept a width from the form definition? Should it override the normal width of that component?